Formula/easy-rsa.rb

```diff
@@ -0,0 +1,34 @@
+class EasyRsa < Formula
+  desc "CLI utility to build and manage a PKI CA."
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/3.0.1.tar.gz"
+  sha256 "a1fff75a27ea7da3f37fbfed715633f55b9ca25f5b14cac38e525c5c995e68ae"
+
+  depends_on "openssl"
+
+  def install
+    # We don't use the installer from the package, as it doesn't organize things
+    # well for brew install. It puts the binary in the prefix root, and defaults
+    # to a pki path that will be overwritten on installation. Instead, I rewrite
```

Comment:
Let's compact this a bit. Can probably live without the:
```
Instead, I rewrite its defaults to find its SSL and PKI in brew-appropriate locations.
```
part, because again it should be obvious enough.